profiles/common.nix

```diff
@@ -4,7 +4,8 @@
 
   imports = [
     ./security.nix
-    ./telegraf
+    ../services/telegraf
+    ./zfs.nix
```

Comment:
Build02 is not using zfs. It is using ext4.

Reply:
Yes. It's just a noop in this case.